internal/xds/util.go

```diff
@@ -38,3 +61,158 @@ func ClusterLoadAssignmentName(service types.NamespacedName, portName string) st
 
 	return strings.Join(name, "/")
 }
+
+// TypeMapping maps xDS type URLs from v2 to v3.
+var TypeMapping map[string]string
+
+func init() {
```

Comment:
Nit: Why do this one on `init()` rather than statically? Maybe might be worth a comment as well for future us.

Reply:
Originally I had something like `var TypeMapping = build_table()`. I switched to `init`, but I don't think there's much practical difference. Happy to take whichever approach is preferred :)

Reply:
I'll leave this as it for now; we will remove the code in Contour 1.11 when we are fully over to v3.